include/galsim/SBInclinedExponentialImpl.h

```diff
@@ -0,0 +1,218 @@
+/* -*- c++ -*-
+ * Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+ * https://github.com/GalSim-developers
+ *
+ * This file is part of GalSim: The modular galaxy image simulation toolkit.
+ * https://github.com/GalSim-developers/GalSim
+ *
+ * GalSim is free software: redistribution and use in source and binary forms,
+ * with or without modification, are permitted provided that the following
+ * conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions, and the disclaimer given in the accompanying LICENSE
+ *    file.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions, and the disclaimer given in the documentation
+ *    and/or other materials provided with the distribution.
+ */
+
+#ifndef GalSim_SBInclinedExponentialImpl_H
+#define GalSim_SBInclinedExponentialImpl_H
+
+#include "SBProfileImpl.h"
+#include "SBInclinedExponential.h"
+#include "LRUCache.h"
+#include "Table.h"
+
+namespace galsim {
+
+    /// @brief A private class that caches the needed parameters for each inclined exponential angle 'i'
+    class InclinedExponentialInfo
+    {
```

Comment:
I left this in in case we want to enable photon-shooting at a later date, in which case caching would be useful (as it would require doing an IFT and then interpolating based off of the results to get x values). I could remove the use of this class, but that might be a waste of time if it'll just be reimplemented later